docs/ModelZoo.md

```diff
@@ -0,0 +1,1024 @@
+# NNCF Compressed Model Zoo
+
+Here we present the results achieved using our sample scripts, example patches to third-party repositories and NNCF configuration files.
+
+- [PyTorch](#pytorch)
+  * [Classification](#pytorch-classification)
+  * [Object Detection](#pytorch-object-detection)
+  * [Semantic Segmentation](#pytorch-semantic-segmentation)
+  * [Natural Language Processing (3rd-party training pipelines)](#pytorch-nlp-huggingface-transformers-powered-models)
+- [TensorFlow](#tensorflow)
+  * [Classification](#tensorflow-classification)
+  * [Object Detection](#tensorflow-object-detection)
+  * [Instance Segmentation](#tensorflow-instance-segmentation)
+- [ONNX](#onnx)
+
+## PyTorch
+
+### PyTorch Classification
+
+<table>
+    <thead>
+		<tr>
+			<th style="text-align: center;" colspan="6"><br>Quantization<br>&nbsp</th>
+		</tr>
+		<tr>
+			<th style="text-align: center;">Model</th>
+			<th style="text-align: center;">Compression algorithm</th>
+			<th style="text-align: center;">Dataset</th>
+			<th style="text-align: center;">Accuracy (<em>drop</em>) %</th>
+			<th style="text-align: center;">NNCF config file</th>
+			<th style="text-align: center;">Checkpoint</th>
+		</tr>
+	</thead>
+	<tbody align="center">
+		<tr>
+			<td style="text-align: center;">ResNet-50</td>
```

Comment:
`style="text-align: center;"` is not needed. 
`<tbody align="center">` is enough.

Reply:
Done